Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -88,9 +121,9 @@ def render_GET(self, request):
         """
         self.events_request = request
 
-        request.write(json.dumps({"type": "events_start"}))
+        request.write(json.dumps({"type": "events_start"}) + '\n')
```

Comment:
No it's not. Do you have a better suggestion how to parse an incoming string without separator between the events? So something like:

```
{"type": "event1"}{"type": "event2"}
```